airbyte-integrations/base/base.sh

```diff
@@ -51,6 +51,9 @@ function main() {
     # todo: state should be optional: --state "$STATE_FILE"
     eval "$AIRBYTE_READ_CMD" --config "$CONFIG_FILE" --catalog "$CATALOG_FILE"
     ;;
+  write)
```

Comment:
we should pass in an env var for source/dest so only one of read and write are valid at this level and show an error if the wrong one is called.

Reply:
agreed. let's do this in a separate PR.